modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/SystemObject1toMObjectRelatedModelsProviderImpl.java

```diff
@@ -211,12 +211,16 @@ public int getRelatedModelsCount(
 	@Override
 	public List<T> getUnrelatedModels(
```

Comment:
@mateussgn do you know why we're implementing this method just for System Objects RelatedModelsProvider?

Reply:
@gabrielwas  Because in the `ItemSelectorView` of a custom object, the unrelated entries is fetch by the rest layer method. But for the `ItemSelectorView` of a system, she use this method